src/Symfony/Component/Notifier/Message/EmailMessage.php

```diff
@@ -84,11 +84,17 @@ public function getSubject(): string
         return '';
     }
 
+    /**
+     * @return null
```

Comment:
Should be removed, this is an implementation detail, the return type is the only contract here.